crates/relayer-cli/src/commands/evidence.rs

```diff
@@ -507,8 +516,10 @@ fn submit_light_client_attack_evidence(
         counterparty.id(),
     );
 
+    let consumer_id = fetch_ccv_consumer_id(counterparty, &counterparty_client_id)?;
```

Comment:
This might happen.
What do you think of this updated `submit_light_client_attack_evidence()` https://gist.github.com/ljoss17/0b253663b317b76061be170b60581143 ?

Reply:
Looks good I think. Can you commit? It will be easier to review.

Reply:
Sure, done in 54fdd9a75935c2bcc0f0053051893e6794d310b8